src/Bravo/Signer.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <stdexcept>
+
+#include "Transaction.h"
+#include "../Data.h"
+#include "../Hash.h"
+#include "../PrivateKey.h"
+
+namespace TW::Bravo {
+
+/// Helper class that performs Bravo transaction signing.
+class Signer {
+public:
+    const Data chainID;
+
+    /// Initializes a signer with a chain identifier.
+    Signer(const Data& chainID) : chainID(chainID) {}
+
+    /// Signs the given transaction.
+    void sign(const PrivateKey& privateKey, Transaction& transaction, int(*canonicalChecker)(uint8_t by, uint8_t sig[64]) = is_canonical) const;
+
+    /// Verifies the given transaction.
+    bool verify(const PrivateKey& privateKey, const Signature& signature, Transaction& transaction) const noexcept;
```

Comment:
It seems this one is not used, you mentioned to remove it?

Reply:
Ah! Missed. will remove it.